docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -3781,6 +3781,16 @@ kafka:
   # CLI flag: -ingest-storage.kafka.write-clients
   [write_clients: <int> | default = 1]
 
+  # The username to use to authenticate to Kafka using SASL plain mechanism.
+  # SASL is enabled when both the username and password have been configured.
```

Comment:
```suggestion
  # To enable SASL, configure both the username and password.
```

Reply:
We try to avoid passive voice in docs.

Reply:
Done in https://github.com/grafana/mimir/pull/9584/commits/1b51166f8d35f5f67bbf8c575b999e03ce0276db